tools/karma_template.conf.js

```diff
@@ -115,7 +115,7 @@ module.exports = function(config) {
         flags: ['--blacklist-accelerated-compositing', '--blacklist-webgl']
       },
       chrome_webgpu: {
-        base: 'Chrome',
+        base: 'ChromeCanary',
```

Comment:
Do we need to do similar change here https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgpu/karma.conf.js#L71 ?

Reply:
Is that still in use? I don't find a counterpart in cpu and webgl backend.

Reply:
> Is that still in use? I don't find a counterpart in cpu and webgl backend.

@haoyunfeix may have more knowlege about it. I think it's still used by the `yarn karma start --browsers=chrome_webgpu` as the workaround in windows to run the yarn test.

Reply:
Yes, that also needs the same change.
But there maybe an important problem for WSL bazel test. The `.bazelrc.user` file needs change CHROME_BIN to CHROME_CANARY_BIN for WebGPU backend, but other backends still needs CHROME_BIN.
We may workaround in bazel test command, but I don't think it's a good idea.
https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md 
```
run --test_env=CHROME_BIN --test_env=WSLENV --test_env=WSL_DISTRO_NAME --define DISPLAY=:0
test --test_env=CHROME_BIN --test_env=WSLENV --test_env=WSL_DISTRO_NAME --define DISPLAY=:0"
```

Reply:
Then what's the problem of `yarn test` in WSL? And why do we need WSL anyway?

Reply:
On my WSL env, `yarn test` will stop by a warning:
```
20 05 2022 07:11:11.204:ERROR [launcher]: No binary for ChromeCanary browser on your platform.
  Please, set "CHROME_CANARY_BIN" env variable.
```
We use WSL because bazel rules for karma [karma_web_test ](https://bazelbuild.github.io/rules_nodejs/Karma.html#karma_web_test)does not support Windows, you can refer issue https://github.com/tensorflow/tfjs/issues/5660

Reply:
Thanks, so is `yarn karma start --browsers=chrome_webgpu` still necessary? If no, then `tfjs-backend-webgpu/karma.conf.js` is obsolete, right?

Reply:
I still use it in windows. I rarely use wsl ;(. Can we keep it for a while?

Reply:
So `yarn karma start --browsers=chrome_webgpu` is for Windows without WSL? I think I'm on track now.

Reply:
>So yarn karma start --browsers=chrome_webgpu is for Windows without WSL?

Yes.